clar2wasm/tests/wasm-generation/noop.rs

```diff
@@ -0,0 +1,27 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use proptest::proptest;
+use proptest::strategy::Strategy;
+
+use crate::{int, uint, PropValue};
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crossprop_noop_to_int(val in uint().prop_map(PropValue)) {
+        crosscheck_compare_only(
+            &format!("(to-int {val})")
+        )
```

Comment:
This is extremely easy to find out what the end answer should be, so this should use `crosscheck`.

Reply:
What if we mess up something in the crosscheck function, and both the compiler and interpreted version returns `Err` constantly? The test shouldn't pass in my opinion.

Reply:
Ok. Makes sense. Changed function implementation.